### ISSUE TITLE
MCH wi

### ISSUE THREAD

**Issue by elmigo69** [Nik-Potokar/XIVSlothCombo#1481]
![image](https://github.com/Nik-Potokar/XIVSlothCombo/assets/119017196/2eea7d19-3c4a-4b98-b888-f40e55663527)

With the change to barel stabilizer, the conditional for wilfire to be at or above 50 heat is not right as both barel stabilizer qand wildfire have de same 2 min cd, you will always execute them toghether on an hypercharge window (sorry for eng, not my native language)

**Comment by Kagekazu** [comment]
Hello! 

i forgot to adapt the layout, but codewise it does : 
- If Barrel stabiliser if off cooldown and u selected the option, it will use it.
- if u have 50 heat or the buff from barrel stabiliser it will do hypercharge
- if u have the lvl for hypercharge + the lvl for wildfire and both are off cooldown, if ur last action was hypercharge it will use wildfire
- if u have gauss round/ ricochet selected, it will weave it in between every heatblast

but i adapted the text so it represents it better.

![image](https://github.com/Nik-Potokar/XIVSlothCombo/assets/95875972/b2703926-dd7c-4f29-83b3-61e5e81574d9)

**Comment by elmigo69** [comment]
you are the best thank's

**Comment by elmigo69** [comment]
I might not doing it right but it seems to me that wildfire isnt use in the combo if not at or above 50 heat, even if barel and/or overheated is on - the text is not changed too, mybe i have to do something to updtae it?

**Comment by Kagekazu** [comment]
yes, it will be once the above PR is merged into main
( PR #1434 )
